docs/ldml/tr35-dates.md

```diff
@@ -877,6 +877,8 @@ For the purposes of determining a repeating field, standalone fields and format
 
 In case there is no matching pattern, the intervalFormatFallback defines the fallback pattern. The fallback pattern is of the form "{0} - {1}" or "{1} - {0}", where {0} is replaced by the start datetime, and {1} is replaced by the end datetime. The fallback pattern determines the default order of the interval pattern. "{0} - {1}" means the first part of the interval patterns in current local are formatted with the start datetime, while "{1} - {0}" means the first part of the interval patterns in current locale are formatted with the end datetime.
 
+For the patterns in both `greatestDifference` and `intervalFormatFallback` elements, literal text that includes ASCII-range letters A-Za-z must be enclosed in single straight quotes ('). As with `dateTimeFormat` elements, text in these patterns (other than substitution markers {0} and {1) is interpreted as part of a date/time pattern, and is subject to the same rules described in [Date Format Patterns](#Date_Format_Patterns).
```

Comment:
I agree with Mark on not changing the spec until we get our story straight for CLDR-10321.